release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -194,6 +194,11 @@ With this update, you can customize workload notifications on the *User Preferen
 ==== Improved quota visibility
 With this update, non-admin users are now able to view their usage of the `AppliedClusterResourceQuota` on the *Project Overview*, *ResourceQuotas*, and *API Explorer* pages in order to determine the cluster-scoped quota available for use. Additionally, `AppliedClusterResourceQuota` details can now be found on the *Search* page.
 
+[id="ocp-cluster-support-level"]
+==== Cluster support level
+
+{product-title} now allows you to easily view support level information about your cluster on the *Overview* -> *Details* card, in the *Cluster Settings*, in the *About* modal, and adds a notification to your notifications drawer when the cluster is unsupported. You can manage subscription settings under the *Service Level Agreement (SLA)* of the *Overview* page.
```

Comment:
```suggestion
{product-title} now enables you to view support level information about your cluster on the *Overview* -> *Details* card, in the *Cluster Settings*, in the *About* modal, and adds a notification to your notifications drawer when the cluster is unsupported. You can manage subscription settings under the *Service Level Agreement (SLA)* of the *Overview* page.
```

- From the IBM Style Guide on "allow"

> allow   verb
> Avoid stating that inanimate objects grant abilities to people, as in "the product allows you to...". Whenever possible, use a direct, user-focused alternative such as "you can use the product to..." or "with this product, users can..."

-  "Easy" is a subjective term, I would try to avoid it. IBM Style Guide says:

> Use with caution. It is normally not accurate to say, “With IBM's service products, installation of your new POWER system will be easy.” It may be accurate to say, “IBM's service products can be used to make installation of your new POWER system easier than if you tried to manage your own installation.”

- The first sentence is really hard to follow, in my opinion. I would consider something like: 

"{product-title} now enables you to view support level information about your cluster. You can add a notification to your notifications drawer when the cluster is unsupported. To add a notification, navigate to *Overview* -> *Details* card in the *Cluster Settings*, in the *About* modal. From the Overview page, you can manage subscription settings under the *Service Level Agreement (SLA)*. "